extensions/adobe/experience/offer-management/fallback-offer.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/fallback-offer",
+  "meta:extends": [
+    "https://ns.adobe.com/experience/offer-management/offer"
+  ],
+  "type": "object",
+  "title": "Adobe Experience Cloud Fallback Offer",
+  "description": "Adobe Experience Cloud Fallback Offer",
+  "definitions": {
+    "fallback-offer": {
+      "type": "object",
+      "properties": {
+        "@id": {
+          "type": "string",
+          "format": "uri",
+          "title": "ID",
+          "description": "The unique identifier of the offer"
+        },
+        "xdm:name": {
+          "type": "string",
+          "title": "Name",
+          "description": "Offer name"
+        },
+        "xdm:status": {
+          "type": "string",
+          "title": "Status",
+          "description": "Offer Status",
+          "enum": [
+            "draft",
+            "pending",
+            "rejected",
+            "approved",
+            "inUse",
+            "archived"
+          ],
+          "meta:enum": {
+            "draft": "Draft",
+            "pending": "Pending",
+            "rejected": "Rejected",
+            "approved": "Approved",
+            "inUse": "In Use",
+            "archived": "Archived"
+          }
+        },
+        "xdm:representations": {
+          "type": "array",
+          "items": {
+            "$ref":
+              "https://ns.adobe.com/experience/offer-management/offer-representation"
+          },
+          "title": "Representations",
+          "description": "Offer representations for different placements"
+        },
+        "xdm:metadata": {
```

Comment:
XDM is metadata - this doesn't make any sense here...

Reply:
See `xdm:customMetadata` at https://github.com/adobe/xdm/pull/245/files#diff-d7d942db7b99d2d903bd73ead182ad8cR66